components/DocumentViewer/index.ts

```diff
@@ -0,0 +1 @@
+export { default } from './DocumentViewer';
```

Comment:
is this necessary? other components such as `Chatbar` do not have an `index.ts`, I wonder what the standard practice is

Reply:
not sure

Reply:
its ok, we can keep for now and easily remove later, I don't think its necessary fwiw but not sure why some of the components have it and some don't